node/src/main/scala/org/bitcoins/node/networking/P2PClient.scala

```diff
@@ -78,8 +84,12 @@ case class P2PClientActor(
       self.forward(networkMsg)
     case message: Tcp.Message =>
       val newUnalignedBytes =
-        handleTcpMessage(message, Some(peer), unalignedBytes)
+        Await.result(handleTcpMessage(message, Some(peer), unalignedBytes),
```

Comment:
Is the purpose of this `Await` to get rid of the compiler warning? In that case, this is better IMO:


```scala
   case connected: Tcp.Connected =>
      val _ = handleEvent(connected, unalignedBytes = ByteVector.empty)
      ()
```